frontend/packages/dsco/README.md

```diff
@@ -6,7 +6,6 @@ Welcome to the DSCO Design System! This repository contains the design system us
 
 - [Introduction](#introduction)
 - [Getting Started](#getting-started)
-- [Installation](#installation)
```

Comment:
is this intended?

Reply:
Yeah, this section doesn't exist in the README right now.